.github/workflows/mynmr_tools_ci.yml

```diff
@@ -73,7 +73,37 @@ jobs:
           cd clients/java
           mvn compile
           mvn test
-      
+
+  # JavaScript Client (latest Node version)
+  javascript-latest-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Javascript client test
+        run: |
+          cd clients/js
+          npm ci
+          npm run test
+
+  # JavaScript Client (lower npm version)
+  javascript-older-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
```

Comment:
This is insufficient to set the JavaScript version.  Note that the `npm install -g npm@5.7.1` only installs npm; it does not install Node.js.

Reply:
The oldest version I could specify was:
        with:
          node-version: 8.x
With this plus 'npm install -g npm@5.7.1'  the command 'npm version' showed:
npm: 5.7.1
node: 8.17.0
The default (not node version specified) yields: node: 10.24.1.
When using
with:
     node-version: 7.x
or older the command 'npm ci' is not working.
Version 8.x old enough?

Reply:
Hmm, after I posted the above comment I saw Elango's changes. Tried again with '6.x' in my development version and now it also works, i.e. 'npm ci' and subsequent 'npm run test' run fine. Still puzzled. Back when it did not work for me I used 'npm help' and it indeed did not list the ci command.

Reply:
To close the loop: this is fixed in https://github.com/google/myanmar-tools/pull/86